docs/java/how-to-set-a-schedule-to-start-timeout-in-java.md

```diff
@@ -0,0 +1,41 @@
+---
+id: how-to-set-a-schedule-to-start-timeout-in-java
+title: How to set a Schedule-To-Start Timeout in Java
+sidebar_label: Schedule-To-Start Timeout
+description: To set a Schedule-To-Start Timeout, use `ActivityOptions.newBuilder.setScheduleToStartTimeout​`].
+tags:
+  - Java
+  - how-to
+  - developer-guide
+---
+
+To set a [Schedule-To-Start Timeout](/docs/concepts/what-is-a-schedule-to-start-timeout), use [`ActivityOptions.newBuilder.setScheduleToStartTimeout​`](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/activity/ActivityOptions.Builder.html).
+
+- Type: `Duration`
+- Default: Unlimited. This timeout is non-retryable.
```

Comment:
> This timeout is "non-retryable" 

While, one could reason about this by understanding what this timeout does...
This is worth calling out at the context level + at the concept page itself - which I don't think it is currently.

Reply:
👍